ocaml/testsuite/tests/typing-layouts/erasable_annot.ml

```diff
@@ -0,0 +1,241 @@
+(* TEST
+   * expect
+   flags = "-extension layouts -only-erasable-extensions"
```

Comment:
Shouldn't need `-extension layouts` I think.

Reply:
Yep, removed the flag